src/components/BookList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Books from './Books';
+
+const BookList = () => {
+  const books = [
+    {
+      id: 1,
+      genre: 'Action',
+      title: 'The Hunger Games',
+      author: 'Suzanne Collins',
+      progress: '64%',
+      chapter: 'Chapter 17',
+    },
+    {
+      id: 2,
+      genre: 'Fiction',
+      title: 'Dune',
+      author: 'Frank Herbert',
+      progress: '8%',
+      chapter: 'Chapter 3: "A Lesson Learned"',
+    },
+  ];
+
+  return (
+    <div>
+      <div>
+        {books.map((book) => (
+          <Books
+            key={book.id}
+            genre={book.genre}
+            title={book.title}
+            author={book.author}
+            progress={book.progress}
+            chapter={book.chapter}
+          />
+        ))}
+      </div>
+    </div>
+  );
+};
```

Comment:
Though you did a great job here, the books are not being displayed on the UI. Have a look at the screenshot. ⬇️
![image](https://github.com/MarcoDDM/bookstore/assets/33268927/9816f6db-5549-47b6-b8d5-b06c38f8fe51)